src/services/logger.ts

```diff
@@ -40,16 +40,80 @@ const attemptFaroWarn = (msg: string, context?: LogContext) => {
   }
 };
 
-const attemptFaroErr = (err: Error | unknown, context?: LogContext) => {
+/**
+ * Checks unknown error for properties from FetchError and adds them to the context
+ * Property names from the error are prepended with an underscore when added to the error context sent to Faro
+ * Note this renames the "message" to "_errorMessage" in hopes of reducing conflicts of future use of that property name
```

Comment:
Do we really need to care about this?

Reply:
Probably not no 🤷

Reply:
but if we're really digging into what came from where it might speed things up a bit